pkg/services/ngalert/api/compat.go

```diff
@@ -456,7 +456,17 @@ func NotificationSettingsFromAlertRuleNotificationSettings(ns *definitions.Alert
 	}
 }
 
-func ApiRecordFromModelRecord(r *models.Record) *definitions.Record {
+func AlertRuleRecordFromRecord(r *definitions.Record) *models.Record {
```

Comment:
Looks like we both wrote the same conversion func :D

I think this can be dropped (or can replace) `ModelRecordFromApiRecord` below

Reply:
Just dropped this, prefer your naming scheme there 👍